docs/examples/using-terraform-with-google-cloud.md

```diff
@@ -0,0 +1,130 @@
+---
+description: This guide shows you how to use Semaphore to set up a pipeline using Terraform with Google Cloud.
+---
+
+# Using Terraform with Google Cloud
+
+This guide shows you how to use Semaphore to set up a pipeline
+using Terraform with Google Cloud.
+
+For this guide you will need:
+
+- [A working Semaphore project][create-project] with a basic CI pipeline.
+You can use one of the documented [use cases][use-cases] or
+[language guides][language-guides] as a starting point.
+- Basic familiarity with Git and SSH.
+
+## Store credentials in secrets
+
+- Store the Terraform Personal Access Token
+
+```bash
+sem create secret terraform-token \
+  -e TERRAFORM_TOKEN=<your-personal-token> 
+```  
+
```

Comment:
This token was only needed for Digitalocean, the token is created on Digitalocean UI (web) to have a private api-key.
For gcloud only the ssh key and the gcloud.json is needed.
To be able to use the private key on gcloud, the public part must be added to the google-cloud-project 'Metadata/SSh keys'

Reply:
How does it look now? btw is it not necessary to run `ssh-keyscan` or `ssh-add` to make the connection work without any confirmation from our side?